assets/scss/thema/dies.scss

```diff
@@ -1,72 +1,236 @@
-$kleur-primair: #903921;
-$kleur-secundair: #c8611a;
-$kleur-textheader: #d08976;
-$kleur-text: #fffbfa;
-$kleur-hover: #e9600b;
+// $kleur-primair: #903921;
```

Comment:
Je kan gecommente code gewoon verwijderen, je kan het toch terug vinden in de got history